### ISSUE TITLE
Unable to open config UI in v0.2

### ISSUE THREAD

**Issue by edbrannin** [jparnell8839/AlwaysOnTop#3]
I just did this:

1. Open the Config panel for AlwaysOnTop v0.1
2. Quit AlwaysOnTop v0.1 (from the control panel or context menu, I don't remember)
3. Downloaded AlwaysOnTop v0.2 from the git repo here
4. Right-click on the new purple Tray icon
5. Select "Always On Top"

Expected result:

Shiny new config UI (hopefully with the disappeared-from-the-menu "Launch at Startup" checkbox)

Observed result:

Nothing happens (except the context-menu from step 4 closes).

**Comment by edbrannin** [comment]
Also, one of my Chrome windows is stuck in front of everything else, even after quitting 0.2 -- this is not something my rules were set up for under 0.1.

**Comment by jparnell8839** [comment]
Windows being stuck in "Always on Top" after exiting was something I just noticed today; there may have been an update to AutoIt's API that made this stick even when an AutoIT executable wasn't running. This is what prompted me to change the method of closing the program to make sure that windows weren't stuck on top if it was running. Unfortunately, if the program crashes (for whatever reason) or is killed from command line / task manger, it may still be stuck.

I also accidentally uploaded a compiled "debugging" version of it, where I was testing the new exit function. You may have snagged that version before I re-uploaded the correct one. This would explain why Chrome was still stuck on top even after program exit

However, I have never had a config UI, or a "Launch at Startup" option. Implementing an "Options" UI was something that was only suggested today (see the other Issue on here by a different user), and that I plan to implement in the near future.

It may be that someone took my source and made their own with added functionality; it's their right to do so, this is licensed under GPL. However, if you still have the other version that had these options, could you attach it to a reply? I'd like to see if it is derived from mine

On another note, I also just released 0.2.1, which changes the mouse cursor to the cross while it's waiting for you to pick the window you want to remain on top. Please try it out and let me know what you think

**Comment by edbrannin** [comment]
Whoops.  I probably posted #1 to the wrong repo because, y'know, it didn't actually tell me where to find it. :)

Here's the one I've been using.

[AlwaysOnTop.zip](https://github.com/jparnell8839/AlwaysOnTop/files/652842/AlwaysOnTop.zip)

**Comment by jparnell8839** [comment]
Yea, what you sent was not my code. After decompiling it, it appears to be wrote in Visual C#. I didn't explore anything past that, because frankly I have no use in stealing someone else's code and I don't know what the license from his is.

However, it appears as if our programs operate similarly under the hood. From running his program, I can see that you filter window names by the titlebar value, and then trigger it to always be on top. The key differences are his is persistent (as expressed the config file in the zip you submitted) while mine is triggered each time you run the program, and he has configurable options, while I do not. The fact that we have the same name for our program is kind of comical; I developed mine in 2012, but the repo I originally hosted to was taken offline, so I moved to GitHub earlier this year.

However, if you cannot find the developer of the version you have, and have feature suggestions, create issues here (one issue per suggestion), and I will implement as I have time.

In the meantime, I'm going to close this one, since we're obviously talking about two different programs with the same title
